Dockerfile

```diff
@@ -51,10 +51,10 @@ RUN julia -e 'Pkg.add("RDatasets")'
 # R installation
 RUN echo 'R_LIBS_USER=/home/jovyan/.R:/usr/lib/R/site-library' > /home/jovyan/.Renviron
 RUN echo 'options(repos=structure(c(CRAN="http://cran.rstudio.com")))' > /home/jovyan/.Rprofile
-RUN mkdir /home/jovyan/.R/
+RUN mkdir /home/jovyan/.R/; echo "PKG_CXXFLAGS = '-std=c++11'" > ~/.R/Makevars
```

Comment:
I'll go ahead and move this with the other R dotfiles above in a separate commit after I merge.